fjord/feedback/static/js/cards.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * cards expects:
+ *
+ * - all cards are divs with class "card"
+ * - inactive cards have the "inactive" class; when starting out
+ *   all cards should be marked inactive but the first one
+ * - the back button has the "back-button-container" class
+ * - data-focus attribute on the card has the id of the element
+ *   that should receive focus
+ * - header -> h1 gets the value of the data-title attribute on
+ *   the card
+ */
```

Comment:
This sucks. It should probably be a class where the above things are configurable, but I was trying to minimize the number of changes I was doing so I opted to document it for now and maybe fix it later.